packages/ui/src/views/repo/repo-summary/repo-summary.tsx

```diff
@@ -210,7 +219,10 @@ export function RepoSummaryView({
                   currentBranch={{
                     ...selectedBranchTag,
                     // TODO: it is required to transfer the real data that the currently selected branch should contain
-                    behindAhead: { ahead: 10, behind: 20 }
+                    behindAhead: {
```

Comment:
Since I initially thought that this data would be coming directly from the branch, I structured the props this way. In this case, we can simplify and update the props and types for the component.

For example:
```tsx
<BranchInfoBar
     defaultBranchName={repository?.default_branch}
     spaceId={spaceId}
     repoId={repoId}
     currentBranchDivergence={currentBranchDivergence}
 />
```

BranchInfoBar component
```tsx
interface BranchInfoBarProps {
  defaultBranchName?: string
  spaceId: string
  repoId: string
  currentBranchDivergence: {
    ahead: number
    behind: number
  }
}

export const BranchInfoBar: FC<BranchInfoBarProps> = ({
  defaultBranchName = 'main',
  spaceId,
  repoId,
  currentBranchDivergence = {
    ahead: 0,
    behind: 0
  }
}) => {
  const { ahead, behind } = currentBranchDivergence
  const hasBehind = behind > 0
  const hasAhead = ahead > 0
```

Reply:
updated with same